packages/plugin/src/theia.d.ts

```diff
@@ -38,6 +38,16 @@ declare module '@theia/plugin' {
 
         static create(func: () => void): Disposable;
 
+        /**
+   * Combine many disposable-likes into one. Use this method
```

Comment:
formatting issue

Reply:
I don't know why, but formatting options of Theia, VsCode or typescript formatter let the document like this...

Reply:
fixed

Reply:
for some reasons ts-lint doesn't check the `theia.d.ts` file